content/english/javascript-basics/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Javascript: Basics"
+description: "Basics of javascript"
+date: 2020-07-29T18:45:38-07:00
+prereq: "none"
+difficulty: "Beginner"
+icon: "fab fa-python"
```

Comment:
Please change the icon to something more appropriate (not the python icon). https://forkaweso.me/Fork-Awesome/icons/